src/librustc_back/target/windows_base.rs

```diff
@@ -63,7 +60,32 @@ pub fn opts() -> TargetOptions {
 
             // Always enable DEP (NX bit) when it is available
             "-Wl,--nxcompat".to_string(),
+
+            // Do not use the standard system startup files or libraries when linking
+            "-nostdlib".to_string(),
```

Comment:
My man page's section for `-nostdlib` begins with this exact sentence \<shrug\>.  I could have used `-nostartfiles` since we are also passing `-nodefaultlibs`, but just to make sure...